readme.md

```diff
@@ -598,7 +598,7 @@
 - [Jupyter](https://github.com/markusschanta/awesome-jupyter#readme) - Create and share documents that contain code, equations, visualizations and narrative text.
 - [FIRST Robotics Competition](https://github.com/andrewda/awesome-frc#readme) - International high school robotics championship.
 - [Humane Technology](https://github.com/engagingspaces/awesome-humane-tech#readme) - Open source projects that help improve society.
-
+- [Electronics Health Records](https://github.c/awesome-ehr-deeplearning) - Deep Learning and EHRs.
```

Comment:
The title here should be the same as your list heading and the description should succinctly explain what EHR and deep learning is.